### DISCUSSION TITLE
Why remove DPM and CNE blobs?

### DISCUSSION THREAD

**Discussion by ghost** [Divested-Mobile/DivestOS-Build#165] [Q&A] [answered]
If it is just because they're blobs I'd rather have them back with the extra functionality

**Comment by SkewedZeppelin** [comment, marked as answer]
They were removed for years, actually some of the earliest removed blobs.

I only added DPM back in hopes they'd fix issues on some multi-SIM devices, and it didn't.

DPM and CNE are a pair afaict.

One of the primary benefits of the deblobber is security: (on end of life devices) these blobs are littered with known security issues, removing the unnecessary ones goes a long way.

And on the note of Wi-Fi calling, do you really want your carrier to know your home IP address or that of any other place you visit?

**Reply by ghost** [reply]
Ok

**Reply by ghost** [reply]
Although, lte+ is nice
